doc/articles/features/working-with-accessibility.md

```diff
@@ -90,13 +90,10 @@ When the value of `WinRTFeatureConfiguration.Accessibility.HighContrast` is chan
 
 ## Known issues
 
-- `Hyperlink` in `TextBlock` is not supported.
```

Comment:
Same here, is this supported from a11y point of view?
I think screen readers are expected to read the text, be able to tell it's hyperlink, and double tapping with a11y focus should open the link. Does it work that way?